apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1/DatastoreClientBuilder.cs

```diff
@@ -27,7 +27,7 @@ public partial class DatastoreClient
         /// Creates a new <see cref="DatastoreClientBuilder"/> instance. This method is primarily present for discoverability.
         /// </summary>
         /// <returns>A new <see cref="DatastoreClientBuilder"/> instance.</returns>
-        public DatastoreClientBuilder CreateBuilder() => new DatastoreClientBuilder();
+        public static DatastoreClientBuilder CreateBuilder() => new DatastoreClientBuilder();
```

Comment:
Agreed that these methods should probably go. They'd never be idiomatically used.

Reply:
We can always add them later, of course. They'd really only be present for discoverability.